src/dns-link-labels.ts

```diff
@@ -1,13 +1,35 @@
+import { CID } from 'multiformats/cid'
```

Comment:
full copy of dns-link-labels from https://github.com/ipfs-shipyard/helia-service-worker-gateway/blob/f3229d7568287878c1b89326660b9dfe0b55c94a/src/lib/dns-link-labels.ts#L31